src/UserManager.ts

```diff
@@ -206,41 +207,37 @@ export class UserManager {
      * Returns promise to notify the opening window of response from the authorization endpoint.
      */
     public async signinPopupCallback(url = window.location.href, keepOpen = false): Promise<void> {
-        try {
-            await this._popupNavigator.callback(url, keepOpen);
-            this._logger.info("signinPopupCallback: successful");
-        }
-        catch (err) {
```

Comment:
why is no longer a catch needed?

Reply:
parity with `signoutPopupCallback`'s behavior which also matches the `*RedirectCallback` and `*SilentCallback` methods. Catching the error here makes it difficult for the popup window to detect an error since the function is `void` return.